packages/hurl/src/runner/predicate.rs

```diff
@@ -495,6 +496,28 @@ fn eval_something(
                 type_mismatch: false,
             }),
         },
+
+        // empty
+        PredicateFuncValue::IsEmpty {} => match value {
+            Value::List(values) => Ok(AssertResult {
+                success: values.len() as i64 == 0,
+                actual: values.len().to_string(),
+                expected: 0.to_string(),
+                type_mismatch: false,
+            }),
+            Value::String(data) => Ok(AssertResult {
+                success: data.len() as i64 == 0,
+                actual: data.len().to_string(),
+                expected: 0.to_string(),
+                type_mismatch: false,
+            }),
+            _ => Ok(AssertResult {
```

Comment:
The `count` filter (https://github.com/Orange-OpenSource/hurl/blob/master/packages/hurl/src/runner/filter.rs#L117) can also take a `Bytes` or `Nodeset` value, we could therefore add them in the `empty` predicate.